src/vectorchord/usage/search.md

```diff
@@ -0,0 +1,72 @@
+# Search
+
+Get the nearest 5 neighbors to a vector
+
+```sql
+SELECT * FROM items ORDER BY embedding <-> '[3,1,2]' LIMIT 5;
+```
+
+## Operators
+
+These operators are used for distance metrics:
+
+| Name  | Description                |
+| ----  | -------------------------- |
+| <->   | squared Euclidean distance |
+| <#>   | negative dot product       |
+| <=>   | cosine distance            |
+
+## Filter
+
+For a given category, get the nearest 10 neighbors to a vector
+```sql
+SELECT 1 FROM items WHERE category_id = 1 ORDER BY embedding <#> '[0.5,0.5,0.5]' limit 10
+```
+
+## Query options
+
+#### `vchordrq.probes`
+    
+- Description: This GUC parameter `vchordrq.probes` controls how the vector space assists in query pruning. The more probes, the more accurate the search, but also the slower it is.
+- Type: integer
+- Default: ``
```

Comment:
Code block is not rendered here. Perhaps it's a bug.

Reply:
I set ` `